electron/libelectron/src/backends/photon.cc

```diff
@@ -1,6 +1,5 @@
 #include <dlfcn.h>
 #include <string.h>
-
```

Comment:
Let's try to minimize random whitespace changes

Reply:
Sorry about that. Reverted it